CHANGELOG.rst

```diff
@@ -55,31 +58,14 @@ _____
 * correcting the KeyError
 
 
-[1.0.1] - 2021-02-18
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Added
-_____
-
-* Add new payment processor (ednx-pp).
-
-
 [1.0.0] - 2021-02-09
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Added
 _____
 
-* Reset setting capability.
-
-
-[0.3.1] - 2020-12-28
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Fixed
-_____
+* Add new payment processor (ednx-pp).
 
-* Remove api key from parameters dictionary in Payu.
```

Comment:
I think the version 0.3.1 doesn't exist, however this fix was made so you can include this in the version 1.0.0 changes

Reply:
It was commented on in CHANGELOG but when you see the git log you never see the bumpversion, much less the tag.